osu.Game.Rulesets.Mania/Objects/Drawables/DrawableHoldNote.cs

```diff
@@ -54,24 +115,151 @@ public override Color4 AccentColour
                     return;
                 base.AccentColour = value;
 
-                headPiece.AccentColour = value;
+                tickContainer.Children.ForEach(t => t.AccentColour = value);
+
                 bodyPiece.AccentColour = value;
-                tailPiece.AccentColour = value;
+                head.AccentColour = value;
+                tail.AccentColour = value;
             }
         }
 
         protected override void UpdateState(ArmedState state)
         {
         }
 
-        protected override void Update()
+        /// <summary>
+        /// Handles key down events on the body of the hold note.
+        /// </summary>
+        /// <param name="state">The input state.</param>
+        /// <param name="args">The key down args.</param>
+        /// <returns>Whether the key press was handled.</returns>

```

Comment:
seal and don't xmldoc? feels weird to have this xmldoc in the first place...

like i'm all for documenting things, but not just for the sake of documenting. this will never be useful.